app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -153,16 +196,24 @@ private class HomeTileAdapter(
             if (hasFocus) {
                 backgroundResource = R.drawable.home_tile_title_focused_background
                 textColor = Color.WHITE
+                menuButton.visibility = View.VISIBLE
             } else {
                 backgroundResource = 0
                 textColor = Color.BLACK
+                menuButton.visibility = View.GONE
             }
             titleView.setBackgroundResource(backgroundResource)
             titleView.setTextColor(textColor)
         }
     }
 
-    override fun getItemCount() = tiles.size
+    fun removeItemAtPosition(position: Int) {
+        homeTiles.removeAt(position)
+        notifyItemRemoved(position)
+        notifyItemRangeChanged(position, itemCount)
```

Comment:
Mmmm since we removed the long-click listener, could you remove this?

In the future, if you choose not to address a comment, please comment in response so I know why.